app.js

```diff
@@ -16,6 +18,8 @@ app.use(logger('dev'));
 app.use(bodyParser.json());
```

Comment:
What happened to bodyParser ?

Reply:
@seyi-adeleke I am not clear on your question.